File: api/ExamApp.Api/Data/Answer.cs
using System;

namespace ExamApp.Api.Data;

public class Answer
{
    public int Id { get; set; }
    public string Text { get; set; } = string.Empty;  // Şık metni
    public string? ImageUrl { get; set; }  // Şık resimli ise
    
    public int QuestionId { get; set; }
    public Question Question { get; set; }

    public double? X { get; set; }

    public double? Y { get; set; }

    public int? Width { get; set; }

    public int? Height { get; set; }

    public bool IsCanvasQuestion { get; set; } = false;


}

File: ui/src/app/models/answer.ts
export interface Answer {
  id: number;
  index: number;
  text: string;
  imageUrl: string;
  x: number;
  y: number;
  width: number;
  height: number;
  isCanvasQuestion: boolean;
  tag?: string; // Backend tag (A,B,C,D,E)
  order?: number; // Backend order (0,1,2,...)
}
